django/contrib/messages/storage/cookie.py

```diff
@@ -86,6 +86,7 @@ def _update_cookie(self, encoded_data, response):
                 domain=settings.SESSION_COOKIE_DOMAIN,
                 secure=settings.SESSION_COOKIE_SECURE or None,
                 httponly=settings.SESSION_COOKIE_HTTPONLY or None,
+                samesite=settings.SESSION_COOKIE_SAMESITE or None,
```

Comment:
According to the code comment, `SESSION_COOKIE_SAMESITE` can be `'lax'`, `'strict'` or `None`. When `SESSION_COOKIE_SAMESITE` is set to `None`, does this evaluate to `None or None`? If so, I don't think we need the `or None`, right? (Same in `django/contrib/sessions/middleware.py`)